packages/uni_app/lib/view/restaurant/restaurant_page_view.dart

```diff
@@ -257,22 +263,61 @@ class _RestaurantPageViewState extends GeneralPageViewState<RestaurantPageView>
     final menuItems =
         getRestaurantMenuItems(dayOfWeek, restaurant, locale) ?? [];
     return menuItems.isNotEmpty
-        ? RestaurantCard(
-            name: RestaurantUtils.getRestaurantName(
-              context,
-              locale,
-              restaurant.namePt,
-              restaurant.namePt,
-              restaurant.period,
-            ),
-            icon: RestaurantUtils.getIcon(
-              restaurant.typeEn ?? restaurant.typePt,
+        ? GestureDetector(
+            onTap: () {
+              if (restaurant.openingHours.isNotEmpty) {
```

Comment:
GestureDetector already exist in the GenericCard, maybe the RestaurantCard should recevice a function